glossary.md

```diff
@@ -0,0 +1,59 @@
+Zettle Go API Glossary
+=============
+
+<!-- Add the corresponding section if any term start with the heading letter. Add numbers before letters if there are glossary items that start with numbers.-->
+<!-- Order: numbers, then A–Z. For example, 3GPP is placed before glossary items started with letter a. --<
+<!-- Write full sentences when explaining acronyms and concepts. -->
+<!-- For syntax reference, see https://www.markdownguide.org/extended-syntax/. -->
+
+## A
+<dl>
+  <dt>Acquiring bank</dt>
+  <dd>Also known as acquirer. An acquiring bank is a financial institution that authenticates the card holder and authorises the transaction. It makes sure that the card is valid and the card has sufficient funds to complete the transaction. This process is known as clearing.</dd>
```

Comment:
You've mixed two processes here. Authorisation is one thing and clearing is another one.